pkgs/development/libraries/concord/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, curl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "concord";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Cogmasters";
+    repo = "concord";
+    rev = "v${version}";
+    hash = "sha256-+nuVyqA/V/J6DWAeIs9Pv90ry3px1gJsF460qvfSEH8="; # Fill this in based on the fetch error
+  };
+
+  buildInputs = [
+    curl
+  ];
+
+  installFlags = [
+  	"PREFIX=${"out"}"
```

Comment:
IIRC this is basically the default. I think you can safely remove the `installFlags`

Reply:
Then it was the default for configuration. Which is not done here. Okay. Will resolve.